client/app/libs/i18n/I18nSetup.js

```diff
@@ -14,6 +14,7 @@ import it from 'react-intl/locale-data/it';
 import nb from 'react-intl/locale-data/nb';
 import sv from 'react-intl/locale-data/sv';
 import nl from 'react-intl/locale-data/nl';
+import vn from 'react-intl/locale-data/vn';
```

Comment:
Try ```import vn from 'react-intl/locale-data/vi';```

Reply:
@baohouse Omg thanks, I didn't realized vi was the official code and not vn lol